Code Quality/Reviewing a pull request.md

```diff
@@ -0,0 +1,68 @@
+> Optimism is an occupational hazard of programming; feedback is the treatment. - Kent Beck
+
+This guide should help you review code better. A good code review will provide a lot of useful things, but some of them are:
+
+-   a sense of security (that the feature works well),
+-   a learning opportunity for both parties,
+-   architectural check,
+-   code quality check.
+
+## Language of the code review
+
+-   Be concise
+-   Do not ask, advise
+-   Do not assume the level of knowledge
+-   Be open for discussion, but do not bike shed
+-   Recommend fixes instead of pointing out errors
+
+### Good examples:
+
+> this won't work because the function you're calling is `async`.
+
+> this file is placed in the incorrect folder, it should go to `components/shared`
+
+> I'm not sure this does what you want it to do. Looking at the service as a whole you should probably allow the end user to construct this out of exported functions.
+
+### Bad examples:
+
+> can you please update this function documentation as well?
+
+> this is not according to our guidelines
+
+> naming is not correct here
+
+> pls fix.
+
+## When do I review pull requests?
+
+Reviewing a pull request could (and should) take time. This is not to be rushed since you vouch for the quality of the code that's on review.
+
+If you review code for either multiple projects or you work in a larger team, you might want to think about assigning a certain fixed part of the day to do them (e.g. I review pull requests in the morning). Everything that comes during the day is to be reviewed first thing tomorrow. This should not include urgent things that block hot fixes, etc. Communicate this time to your project manager so it can be correctly reflected in your sprint availability.
```

Comment:
This part is weird. You are talking about assigning a certain part of the day for reviewing pull requests and then in the second part you state that this should be done "first thing tomorrow".

I would go in a different direction. When and how often pull requests should be reviewed is up to you and the project team but as a rule of thumb - a pull request shouldn't be unchecked for more than 24 hours. That means that you should review pull requests at least once a day.
Urgent fixes should, of course, be checked as soon as possible.

Reply:
Yeah I agree - this was the idea; and the suggestion was to have as a fixed portion of the day (no matter if it's morning or evening or whatever).

I'll reword this a bit.

Reply:
I cleared it up a bit. Care to give it another read? 758a44f

Reply:
Great, I think you cleared everything up now 👍